resources/definitions/velleman_vertex.def.json

```diff
@@ -0,0 +1,56 @@
+{
+    "id": "velleman_vertex",
+    "version": 2,
+    "name": "Velleman Vertex",
+    "inherits": "fdmprinter",
+    "metadata": {
+        "visible": true,
+        "author": "Cade Cyphers",
+        "manufacturer": "Velleman",
+        "category": "Other",
+        "file_formats": "text/x-gcode",
+        "has_materials": true,
+    },
+    
+    "overrides": {
+        "machine_name": {
+            "default_value": "Velleman Vertex"
+        },
+        "machine_heated_bed": {
+            "default_value": false
+        },
+        "machine_width": {
+            "default_value": 190
+        },
+        "machine_height": {
+            "default_value": 200
+        },
+        "machine_depth": {
+            "default_value": 180
```

Comment:
Going by the specs online, I think Height and Depth are switched around here. In Cura (a little unconventially) the width is the X axis, the depth is the Y axis and the height is the Z axis. It's as if you're looking from the front side of the machine, so to say. Could you verify that this is correct?